src/components/AppBar.jsx

```diff
@@ -101,42 +159,125 @@ export default function AppBar() {
+        <Hidden mdUp>
+          <MobileMenu />
+        </Hidden>
         <Logo />
-        <div className={classes.navigation}>
-          <Container fixed className={classes.menu}>
-            <Typography className={classes.menuItems}>
-              <NavLink to="/dashboard" activeStyle={{ color: activeLinkColor }}>
-                Dashboard
-              </NavLink>
-              <NavLink to="/explore" activeStyle={{ color: activeLinkColor }}>
-                Explore
-              </NavLink>
-              <NavLink to="/search" activeStyle={{ color: activeLinkColor }}>
-                Search
-              </NavLink>
-            </Typography>
-            <div className={classes.search}>
-              <TextField
-                id="search-field"
-                placeholder="Search..."
-                variant="outlined"
-                size="small"
+        <Hidden smDown>
+          <div className={classes.navigation}>
+            <Container fixed className={classes.menu}>
+              <Typography className={classes.menuItems}>
+                <NavLink
+                  to="/dashboard"
+                  activeStyle={{ color: activeLinkColor }}
+                >
+                  Dashboard
+                </NavLink>
+                <NavLink to="/explore" activeStyle={{ color: activeLinkColor }}>
+                  Explore
+                </NavLink>
+                <NavLink to="/search" activeStyle={{ color: activeLinkColor }}>
+                  Search
+                </NavLink>
+              </Typography>
+              <div className={classes.search}>
+                <TextField
+                  id="search-field"
+                  placeholder="Search..."
+                  variant="outlined"
+                  size="small"
+                  InputProps={{
+                    classes: {
+                      input: classes.searchInput
+                    }
+                  }}
+                />
+              </div>
+            </Container>
+          </div>
+        </Hidden>
+        <div>
+          <Hidden smDown>
+            <IconButton>
+              <ChatIcon />
+            </IconButton>
+            <IconButton>
+              <NotificationsIcon />
+            </IconButton>
+          </Hidden>
+          <Hidden smDown>
+            <Button
+              className={classes.accountButton}
+              aria-controls="fade-menu"
+              aria-haspopup="true"
+              endIcon={open ? <ArrowDropUpIcon /> : <ArrowDropDownIcon />}
+              color="inherit"
+              variant="outlined"
+              onClick={handleClick}
+            >
+              Alexander Bergholm
+            </Button>
+          </Hidden>
+          <Hidden mdUp>
+            <IconButton>
+              <AccountCircleIcon
+                aria-controls="fade-menu"
+                aria-haspopup="true"
+                color="inherit"
+                onClick={handleClick}
```

Comment:
Should this onClick go into the parent `IconButton`? I think this is producing a warning in the console for me.

Reply:
Ah, yes! Great catch, Ming!